tests/foreman/cli/test_templatesync.py

```diff
@@ -126,3 +126,25 @@ def test_positive_export_filtered_templates_to_git(self):
 
         :CaseAutomation: NotAutomated
         """
+
+    @tier2
+    def test_positive_export_filtered_templates_to_temp_dir(
+        self, module_org, create_import_export_local_dir
+    ):
+        """Assure templates can be exported to /tmp directory without right permissions
+
+        :id: e0427ee8-698e-4868-952f-5f4723ccee87
+
+        :bz: 1778177
+
+        :Steps: Export the templates matching with regex e.g: `ansible` to /tmp directory.
+
+        :expectedresults: The templates are exported /tmp directory
+
+        :CaseImportance: Medium
+        """
+        dir_path = '/tmp'
+        TemplateSync.exports(
+            {'repo': dir_path, 'organization-id': module_org.id, 'filter': 'ansible'}
+        )
+        assert ssh.command(f'find {dir_path} -type f -name *ansible* | wc -l').stdout[0] == '23'
```

Comment:
```suggestion
        assert ssh.command(f'find {dir_path} -type f -name *ansible*  2>/dev/null| wc -l').stdout[0] == '23'
```

Reply:
We are specially looking into the exact folder `{dir_path}` for modules so I dont think I should redirect the output.

Reply:
I think it would help to avoid the unnecessary error, It's my suggestion only.

Reply:
Yes! But I confirm there wont be any error and if its is I want to see it and failing the test! :)